### ISSUE TITLE
Hardware setup ?

### ISSUE THREAD

**Issue by N0edL** [willumpie82/EsphomeModbusElgaAce#3]
Kan je wat info geven over wat voor hardware je hebt gebruikt en hoe je het hebt aangesloten bedankt.

**Comment by willumpie82** [comment]
Hi, As mentioned on Tweakers.net;

<img width="822" alt="Screenshot 2024-11-02 at 18 32 22" src="https://github.com/user-attachments/assets/bff9bac2-3b3d-4e5c-921f-6583b66841e4">
<img width="822" alt="Screenshot 2024-11-02 at 18 33 05" src="https://github.com/user-attachments/assets/066a9874-7128-4323-9eb5-4da2a52ccb10">

**Comment by willumpie82** [comment]
Also a plain rs485 transceiver should work, but than also mind RX and TX to the esp.

**Comment by N0edL** [comment]
> Hi, As mentioned on Tweakers.net;
> 
> 
> 
> <img width="822" alt="Screenshot 2024-11-02 at 18 32 22" src="https://github.com/user-attachments/assets/bff9bac2-3b3d-4e5c-921f-6583b66841e4">
> 
> <img width="822" alt="Screenshot 2024-11-02 at 18 33 05" src="https://github.com/user-attachments/assets/066a9874-7128-4323-9eb5-4da2a52ccb10">
> 
> 

Ahh oke ik heb een losse rs485 module gekocht en die aangesloten op een esp32 maar dat krijg ik niet werkend

**Comment by N0edL** [comment]
I just wired up a rs485 module to rx2 tx2 and d4. But i dont get anyreading

![image](https://github.com/user-attachments/assets/0a049f25-629d-45ab-b8f9-f78af09cff11)

**Comment by willumpie82** [comment]
> I just wired up a rs485 module to rx2 tx2 and d4. But i dont get anyreading

Did you try this connection?


![](https://community-assets.home-assistant.io/original/4X/3/4/d/34d63a32e88dec4a24963d01637b88e8c754ebdf.jpeg)

**Comment by N0edL** [comment]
Yes i found it it was the flow_control_pin was not on the right one but this is what im logging:
![image](https://github.com/user-attachments/assets/2871b605-9909-4e0e-9ef6-ee7b1ccdc0bc)
Some values dont look good and is this everthring or can i log more ?

**Comment by willumpie82** [comment]
Have a look at my yaml cfg, there are value’s that needs conversion or are a different type

**Comment by N0edL** [comment]
Okay, maar wacht je bent toch nederlands ?

**Comment by N0edL** [comment]
![image](https://github.com/user-attachments/assets/1ac6ec1b-44b5-4dd0-8def-82feac627221)

This is the same as u have but i get -500 degrees

**Comment by willumpie82** [comment]
> Okay, maar wacht je bent toch nederlands ?

Automatisme, op github tik ik over het algemeen engels 😁, consider het een beetje de voertaal aangezien dit een internationaal publiek platform is 😉

**Comment by N0edL** [comment]
Ahh top maar heb jou werk stijl overgenomen en gebruikt en ziet er nu veel beter uit en werkt perfect bedankt voor de reacties en het delen van de code 👍

**Comment by N0edL** [comment]
Nog een kort vraagje wat word bedoeld met Zone1 type, Zone1 functie en App Status 1(b2): backup1 ?

**Comment by N0edL** [comment]
Hey quick question can you also change parameters ?

**Comment by N0edL** [comment]
Ik wil Maximale aanvoertemperatuur, Nadraaitijd, en divalent temperatuur. kunnen aan passen en nog wat cp codes maar ik vind deze niet terug in de lijst
